server/server.go

```diff
@@ -469,43 +469,81 @@ func New(
 	return s, nil
 }
 
-func (s *Server) getHostIP(configIP string) net.IP {
+func (s *Server) getHostIPs(configIP string) []string {
```

Comment:
this should still return `[]net.IP`. IPv6 addresses can be "spelled" in many ways, so it's important to always use the byte form.

Reply:
Alright, fixed :heavy_check_mark: